src/components/Tooltip/index.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import TooltipHolder from './TooltipHolder';
+import TooltipInfo from './TooltipInfo';
+import Tipneed from './Tipneed';
+
+type Props = {
+  info: string,
+};
+
+const Tooltip = (info: Props) =>
```

Comment:
The `info` prop should be destructured.

```suggestion
const Tooltip = ({ info }: Props) =>
```